srcpkgs/grub/template

```diff
@@ -141,19 +147,24 @@ grub-utils_package() {
 		vmove usr/bin/grub-mkfont
 	}
 }
-grub-x86_64-efi_package() {
-	noarch=yes
-	depends="grub>=$version dosfstools efibootmgr"
-	short_desc+=" - x86_64 EFI support"
-	pkg_install() {
-		vmove usr/lib/grub/x86_64-efi
-	}
-}
-grub-i386-efi_package() {
-	noarch=yes
-	depends="grub>=$version dosfstools efibootmgr"
-	short_desc+=" - i386 EFI support"
-	pkg_install() {
-		vmove usr/lib/grub/i386-efi
+
+for _SUPP_BUILD in ${_SUPPLEMENTARY_BUILDS}; do
```

Comment:
i'm sorry but what is this ?

why are you generating package() definitions on the fly ?

can't you just keep them all and just use subpackages= to select which ones are necessary ?

Reply:
To avoid duplicating code but as you wish